tests/test_conversion_imagenet.py

```diff
@@ -754,32 +754,35 @@ def _test_function(self, original_framework, parser):
 
 
 
+
+
+    def test_caffe(self):
+        try:
+            import caffe
+            self._test_function('caffe', self.CaffeParse)
+        except ImportError:
+            print('Please install caffe! Or caffe is not supported in your platform.')
```

Comment:
Raise an error then stop or just print something then skip this test?

Reply:
print to stderr?